bundles/org.eclipse.ui.genericeditor/plugin.xml

```diff
@@ -276,4 +276,13 @@
             label="%genericEditorStrategy">
       </strategy>
    </extension>
+   <extension
+         point="org.eclipse.ui.preferencePages">
+      <page
+            category="org.eclipse.ui.preferencePages.GeneralTextEditor"
+            class="org.eclipse.ui.internal.genericeditor.preferences.GenericEditorPreferencePage"
+            id="org.eclipse.ui.genericeditor.GenericTextEditor"
+            name="%genericEditor_name">
```

Comment:
Form the UI long term strategy point of view: should the category be named "Generic Text Editor**s**" ?

There is a high possibility the other generic text editor (derived editor) preferences may appear here.
Also looking on WebTools (Structured Text Editor**s**) and Text Editor**s** I would also assume it would be more consistent.

![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/964108/b5fdedbd-a742-420b-a4c3-90bf8461a554)